test/utils/IexecWrapper.ts

```diff
@@ -29,6 +29,7 @@ import {
     WorkerpoolRegistry__factory,
     Workerpool__factory,
 } from '../../typechain';
+import { TransferEvent } from '../../typechain/contracts/registries/IRegistry';
```

Comment:
Why not from `../../typechain`?

Reply:
Because there are many:
![image](https://github.com/user-attachments/assets/0c0b2c4f-2256-4ffa-afec-01ad920af7ed)